test/unit/detail/version_check_test.hpp

```diff
@@ -208,85 +195,65 @@ TEST_F(sanity_checks, cookie)
         // ```
         // This should not be tested via EXPECT_EQ(..) because then sharg tests fail if the server wasn't
         // configured correctly and we want to be independent of the server.
-        std::cout << "Cookie:" << std::endl;
+        std::string line{};
+        std::cout << "Cookie:\n";
+
         std::ifstream app_version_file{app_version_filename()};
-        std::string line;
         std::getline(app_version_file, line);
-        std::cout << line << " <-- Should be UNREGISTERED_APP!" << std::endl;
+        std::cout << line << " <-- Should be UNREGISTERED_APP!\n";
+
+        std::cout << app_version_filename().string() << '\n';
+
         std::getline(app_version_file, line);
         std::cout << line << " <-- Should be the latest Sharg version! "
-                  << "Updates done here: https://github.com/OpenMS/usage_plots/blob/master/seqan_versions.txt"
-                  << std::endl;
+                  << "Updates done here: https://github.com/OpenMS/usage_plots/blob/master/seqan_versions.txt\n";
     }
 }
 
 //------------------------------------------------------------------------------
 // version checks
 //------------------------------------------------------------------------------
 
-TEST_F(version_check, option_on)
+TEST_F(version_check_test, option_on)
 {
-    char const * argv[3] = {app_name.c_str(), OPTION_VERSION_CHECK, OPTION_ON};
-
-    auto [out, err, app_call_succeeded] = simulate_parser(3, argv);
+    auto [out, err, app_call_succeeded] = simulate_parser(OPTION_VERSION_CHECK, OPTION_ON, "-f");
 
     EXPECT_EQ(out, "");
     EXPECT_EQ(err, "");
 
     // no timestamp is written since the decision was made explicitly
     if (app_call_succeeded)
-    {
         EXPECT_TRUE(std::filesystem::exists(app_version_filename()));
-    }
     else
-    {
-        std::cout << "App call did not succeed (server offline?) and could thus not be tested.\n";
-    }
+        GTEST_SKIP() << "App call did not succeed (server offline?) and could thus not be tested.";
```

Comment:
```suggestion
        GTEST_SKIP() << "App call did not succeed (server offline?) and could thus not be tested.\n";
```
?

Reply:
Not necessary, macro takes care of formatting